root/etc/e-smith/events/actions/nethserver-tancredi-conf

```diff
@@ -11,3 +11,24 @@ GRANT SELECT on asterisk.ampusers TO 'tancredi'@'localhost' identified by '$db_p
 FLUSH privileges;
 EOF
 
+# set default.ini on first install
+if [[ ! -f /var/lib/tancredi/data/scopes/defaults.ini ]]; then
```

Comment:
Can you set a variable?
```suggestion
dst_file="/var/lib/tancredi/data/scopes/defaults.ini"
src_file="/usr/share/tancredi/data/scopes/defaults.ini"
if [[ ! -f ${dst_file} ]]; then
```

Reply:
I used variable for destination file, not for source because it's used only one time